_layouts/base.html

```diff
@@ -6,13 +6,43 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="description" content="{% if page.description %}{{ page.description }}{% else %}{{ site.description }}{% endif %}">
+
+  <meta content="{{ site.title }}" property="og:site_name">
+  {% if page.title %}
+  <meta content="{{ page.title }}" property="og:title">
+  {% else %}
+  <meta content="{{ site.title }}" property="og:title">
+  {% endif %}
+  {% if page.image %}
+  <meta content="{{ site.url }}{{ site.url }}{{ post.image }}" property="og:image">
```

Comment:
page.image or post.image?

Reply:
Actually `post.image`

Reply:
Ok, but on the line before you query for page.image. Is that intended?

Reply:
Actually it was a typo since we are adding social sharing icons for blogs, it should be `post.image`